.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
this belongs to [global gitignore](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore)

Reply:
Sorry for that, removed it